onnxruntime/core/providers/dnnl/subgraph/dnnl_elementwise.cc

```diff
@@ -35,17 +35,25 @@ void DnnlElementwise::CreatePrimitive(DnnlSubgraphPrimitive& sp, DnnlNode& node)
       }
       break;
     }
+    case dnnl::algorithm::eltwise_soft_relu: {
```

Comment:
are we ever going to get eltwise_soft_relu algorithm for anything other than Softplus?

Reply:
From what I saw in the code, it's also used for sigmoid, that's why we check if the OpType is Softplus